data/values.yml

```diff
@@ -0,0 +1,5 @@
+---
+values:
```

Comment:
1. what about "rule_data" as a top level key. values is a bit generic.
2. but actually, this might be the kind of thing that would lead to conftest throwing errors if it sees the key defined in two places. So we maybe need to invent a convention with a flat top level key structure like this:

```
---
rule_data_for_allowed_registry_prefixes:
- ...
```

Reply:
Maybe double underscore as a delimiter? `ec_rule__allowed_registry_prefixes`  or `rule_data__allowed_registry_prefixes`...

Reply:
`rule_data` is good name. Would it be a bit repetitive to type `data.rule_data` though? That's why I went with `values` which becomes `data.values`. I can easily change it, just double checking that you still like the name in context.

I made a quick test by creating another directory, `data2`, and modifying `make check-release` to also provide this data directory, e.g.:

```diff
diff --git a/Makefile b/Makefile
index dc0d07e..23418ee 100644
--- a/Makefile
+++ b/Makefile
@@ -200,6 +200,7 @@ check-release: ## Run policy evaluation for release
          --namespace $(RELEASE_NAMESPACE) \
          --policy $(POLICY_DIR) \
          --data $(DATA_DIR) \
+         --data $(DATA_DIR)2 \
          --no-fail \
          --output json
```

Then, `cp data/values.yml data2/`. Sure enough, conftest complains about conflicts:
```
🐚 make check-release
Error: running test: load: load documents: 1 error occurred during loading: data2/values.yml: merge error
make: *** [Makefile:199: check-release] Error 1
```

That's expected. Good, collisions are obvious. 

Next, I changed `data2/values.yml` to have a unique (nested) key:
```
🐚 diff data/values.yml data2/values.yml 
3c3
<   allowed_registry_prefixes:
---
>   allowed_registry_prefixes_test:
```
(Notice that both YAML files contain the same top-level key)

Conftest is happy to merge those two YAML files.

Does this address your second point?

There's a scenario that this may be a problem though. If a user decides to use two distinct policy sources which both use a common key but for different purposes, they can only satisfy one. I think the solution to this is that we should really have data sources mapped to a particular policy source, like you suggested in [HACBS-1329](https://issues.redhat.com/browse/HACBS-1329). Wdyt?

Reply:
Oooh, so it does do a "deep" merge? That's interesting. I wasn't sure if it would do that.

Reply:
If conftest is smart enough to merge the keys under the top level key, that does remove my concern, and the motivation for suggesting a flat unique top level key.

I still don't like "values" as a key name though FWIW. I'd say "rule_data" is slightly preferable - even though it's only marginally less generic sounding, we are using it already in the annotations.

Reply:
Changed it to `rule_data`.